package.json

```diff
@@ -117,6 +117,7 @@
     "@types/eslint": "^8.2.2",
     "@types/jest": "^29.5.1",
     "@types/node": "^18.15.11",
+    "@types/rosie": "^0.0.42",
```

Comment:
All the changes in the `graphql-test-utils` directory is just renaming files from `.js` to `.ts` so we don't have typescript errors when using them in the new typescript test files.